### ISSUE TITLE
[Window Terminal]: Keyboard only users are not able to access 'Don't show again', 'Open Settings' and 'X' controls which present inside information pop-up.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11930]
### Windows Terminal version

1.12.2931.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment: **
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps: **
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Inside Startup page, Set Default Terminal to 'Window Console Host'.
4. Save changes the open Terminal app gain.
5. Verify that Keyboard users can access 'Open Settings' and 'X' control.
6. Then open Command palette using 'Ctrl + Shift + p' and in command line mode Execute 'Dir'.
7. Verify that Keyboard users can access 'Don't Show again' and 'X' control.

**User Experience: **
Keyboard users like Keyboard only users, Screen Reader users will be impacted here as they are not able to access control which mouse users are able to access.

**Guideline Reference: **
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

Attachments:
[Keyboard user is not able to access 'Don't Show again', 'Open Settings' and 'X' control.zip](https://github.com/microsoft/terminal/files/7701286/Keyboard.user.is.not.able.to.access.Don.t.Show.again.Open.Settings.and.X.control.zip)

### Expected Behavior

Keyboard only users should be able to access 'Don't show again', 'Open Settings' and 'X' controls which present inside information pop-up. Or A Shortcut should be given or announced to jump directly to Information pop-up.

### Actual Behavior

Keyboard only users are not able to access 'Don't show again', 'Open Settings' and 'X' controls which present inside information pop-up.

Observation: Screen Reader users are only able to access information pop-up in Scan mode.

**Comment by zadjii-msft** [comment]
ideas: 
* How do you tab navigate to the infobar in VS?
* Same question, but with Word? 
* We should consult with the WinUI folks and see what the guidance is here. 
* Last resort, we add an "Alt wormhole" where pressing Alt without any other keys lets you press <kbd>tab</kbd> to get into the tab strip/infobar for tab navigation, and a setting to disable that wormhole as well.
